### PR TITLE
oscdimg: Add version 2.56

### LINKED ISSUE #4665: [Request]: osdcimg
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

osdcimg

### Description

Command-line tool to create an image (.iso) file of a customized 32-bit or 64-bit version of Windows Preinstallation Environment (Windows PE).

### Homepage

https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/oscdimg-command-line-options

### Download Link(s)

https://download.microsoft.com/download/1/f/d/1fd2291e-c0e9-4ae0-beae-fbbe0fe41a5a/adk/Installers/5d984200acbde182fd99cbfbe9bad133.cab

### Some Indication of Popularity/Repute

Googling for "oscdimg" returns 100k hits.

### PR DESCRIPTION
**Pull request by sebthom**
Closes #4665

- [X ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sebthom** [comment]
@rashil2000 I would appreciate if you can have a look at this PR. oscdimg is required by [packer](https://github.com/ScoopInstaller/Main/blob/master/bucket/packer.json) which can be already installed via scoop.

File: bucket/oscdimg.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2.56",
+    "homepage": "https://learn.microsoft.com/windows-hardware/manufacture/desktop/oscdimg-command-line-options",
+    "description": "Command-line tool to create an image (.iso) file of a customized 32-bit or 64-bit version of Windows Preinstallation Environment (Windows PE).",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx"
+    },
+    "url": "https://download.microsoft.com/download/1/f/d/1fd2291e-c0e9-4ae0-beae-fbbe0fe41a5a/adk/Installers/5d984200acbde182fd99cbfbe9bad133.cab",
```
Comment (rashil2000) [inline bucket/oscdimg.json:9]:
```suggestion
    "url": "https://download.microsoft.com/download/1/f/d/1fd2291e-c0e9-4ae0-beae-fbbe0fe41a5a/adk/Installers/5d984200acbde182fd99cbfbe9bad133.cab#/dl.7z",
```

File: bucket/oscdimg.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "hash": "957e11ab029f9e872f08cdd664cd7d206618d51ba716106655ea3d1499ceaf11",
+    "pre_install": [
+        "expand.exe -f:fil720cc132fbb53f3bed2e525eb77bdbc1 \"$dir\\5d984200acbde182fd99cbfbe9bad133.cab\" \"$dir\"",
+        "Rename-Item -Path \"$dir\\fil720cc132fbb53f3bed2e525eb77bdbc1\" -NewName \"oscdimg.exe\"",
+        "Remove-Item -Path \"$dir\\5d984200acbde182fd99cbfbe9bad133.cab\" -Force"
+    ],
```
Comment (rashil2000) [inline bucket/oscdimg.json:15]:
```suggestion
    "pre_install": "Rename-Item -Path \"$dir\\fil720cc132fbb53f3bed2e525eb77bdbc1\" -NewName \"oscdimg.exe\"",
```

**Pull request merged**